ietf-network-inventory.yang

```diff
@@ -96,10 +96,103 @@ module ietf-network-inventory {
         "A physical network element (NE). ";
     }
 
+  /*
+   * Editors' Note: This identity may need to be moved to
+   *                iana-hardware update
+   */
+
+  identity transceivers-module {
+    base ianahw:hardware-class;
+    description
+      "This identity is applicable if the hardware class is some sort
+       of self-contained sub-system which contains one or more
+       transceivers (e.g., optical or electrical transceivers) which
+       are transmitting and receiving a signal from the media.
+       
+       A transceivers-module component can only be contained by a
+       port component.";
+  }
```

Comment:
**[2024-12-11 Base Network Inventory weekly call](https://github.com/ietf-ivy-wg/network-inventory-yang/wiki/minutes-2024-12-11)**

The definition of the transceivers-module is very close to the definition of transponder in optical impairments I-D

- [ ] @italobusi @sergiobelotti @YuChaode Check how an integrated transponder can be modelled in the network inventory model